assets/js/components/user-input/UserInputKeywords.js

```diff
@@ -120,8 +121,17 @@ export default function UserInputKeywords( { slug, max } ) {
 							'googlesitekit-user-input__text-option': values.length > i + 1 || value.length > 0,
 						} ) }
 					>
+						<VisuallyHiden>
+							<label
+								htmlFor={ `${ slug }-keyword-${ i }` }
+							>
+								{ __( 'Enter minimum one (1), maximum three (3) terms', 'google-site-kit' ) }
+							</label>
+						</VisuallyHiden>
 						<TextField
-							label={ i + 1 === values.length ? __( 'Enter minimum one (1), maximum three (3) terms', 'google-site-kit' ) : '' }
+							label={ values.length === 1 && values[ 0 ] === ''
+								? __( 'Enter minimum one (1), maximum three (3) terms', 'google-site-kit' )
+								: '' }
```

Comment:
This change should be removed; it's no longer part of the ACs/IB as we decided it isn't needed anymore. See: https://github.com/google/site-kit-wp/pull/2984/files#r598867190